### PR TITLE
Test Pull Request backup and restore

### PR DESCRIPTION
**Pull request by ralph0211**
A Pull Request to test functionality for backing up and restoring pull requests
images
 More text here just to wrap around the image!

### CODE REVIEW

**Comment by ralph0211** [comment]
Test comment with attachment
images (1)

File: AzurePullRequests.Restore/Enums/RestoreType.cs [hunk 1]
```diff
@@ -1,10 +1,4 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
```
Comment (ralph0211) [inline AzurePullRequests.Restore/Enums/RestoreType.cs:3]: Comment in code with attachment
images

File: AzurePullRequests.Restore/Enums/RestoreType.cs [hunk 2]
```diff
@@ -1,10 +1,4 @@
...
-using System.Text;
-using System.Threading.Tasks;
```
Comment (raph-backrightup) [inline AzurePullRequests.Restore/Enums/RestoreType.cs:5]: Another user comments on a file

**Review by ralph0211 — COMMENTED** [review]
Finished reviewing after one comment

File: AzurePullRequests.Restore/Enums/RestoreType.cs (hunk 2 above)
Comment (ralph0211) [reply AzurePullRequests.Restore/Enums/RestoreType.cs:5]: Different user replies to another user's comment on file

Comment (raph-backrightup) [reply AzurePullRequests.Restore/Enums/RestoreType.cs:5]: Original user replies to same thread with an attachment
istockphoto-1398462038-612x612

**Pull request closed without merging**